<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="21" passed="13" failed="8" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-17T14:05:00 IST" name="OrangeHRM Automation Suite" finished-at="2025-09-17T14:10:52 IST" duration-ms="352086">
    <groups>
    </groups>
    <test started-at="2025-09-17T14:05:00 IST" name="Login Test" finished-at="2025-09-17T14:05:40 IST" duration-ms="39632">
      <class name="com.orangehrm.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.orangehrm.LoginTest@23348b5d]" started-at="2025-09-17T14:05:00 IST" name="setUp" finished-at="2025-09-17T14:05:03 IST" duration-ms="2734" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginDataDrivenTest(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.LoginTest@23348b5d]" started-at="2025-09-17T14:05:03 IST" name="loginDataDrivenTest" data-provider="loginData" finished-at="2025-09-17T14:05:35 IST" duration-ms="31972" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Adm]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception during login for: Adm]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception during login for: Adm
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.LoginTest.loginDataDrivenTest(LoginTest.java:108)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDataDrivenTest -->
        <test-method signature="loginDataDrivenTest(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.LoginTest@23348b5d]" started-at="2025-09-17T14:05:35 IST" name="loginDataDrivenTest" data-provider="loginData" finished-at="2025-09-17T14:05:40 IST" duration-ms="4606" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDataDrivenTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.LoginTest@23348b5d]" started-at="2025-09-17T14:05:40 IST" name="tearDown" finished-at="2025-09-17T14:05:40 IST" duration-ms="276" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.LoginTest -->
    </test> <!-- Login Test -->
    <test started-at="2025-09-17T14:05:40 IST" name="PIM Tests" finished-at="2025-09-17T14:09:07 IST" duration-ms="207431">
      <class name="com.orangehrm.PIMTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:05:40 IST" name="setUp" finished-at="2025-09-17T14:05:50 IST" duration-ms="10295" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addEmployeeTest()[pri:1, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:05:50 IST" name="addEmployeeTest" finished-at="2025-09-17T14:06:11 IST" duration-ms="20361" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmployeeTest -->
        <test-method signature="fillPersonalDetailsTest()[pri:2, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:06:11 IST" name="fillPersonalDetailsTest" finished-at="2025-09-17T14:06:23 IST" duration-ms="12235" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Personal Details test failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Personal Details test failed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.PIMTest.fillPersonalDetailsTest(PIMTest.java:154)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillPersonalDetailsTest -->
        <test-method signature="contactDetailsTest()[pri:3, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:06:23 IST" name="contactDetailsTest" finished-at="2025-09-17T14:06:51 IST" duration-ms="28401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- contactDetailsTest -->
        <test-method signature="emergencyContactsTest()[pri:4, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:06:51 IST" name="emergencyContactsTest" finished-at="2025-09-17T14:07:06 IST" duration-ms="14697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emergencyContactsTest -->
        <test-method signature="dependentsTest()[pri:5, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:07:06 IST" name="dependentsTest" finished-at="2025-09-17T14:07:17 IST" duration-ms="10564" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dependentsTest -->
        <test-method signature="immigrationRecordsTest()[pri:6, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:07:17 IST" name="immigrationRecordsTest" finished-at="2025-09-17T14:07:27 IST" duration-ms="10694" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- immigrationRecordsTest -->
        <test-method signature="jobDetailsTest()[pri:7, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:07:27 IST" name="jobDetailsTest" finished-at="2025-09-17T14:07:39 IST" duration-ms="11910" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jobDetailsTest -->
        <test-method signature="salaryComponentsTest()[pri:8, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:07:39 IST" name="salaryComponentsTest" finished-at="2025-09-17T14:08:04 IST" duration-ms="24630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- salaryComponentsTest -->
        <test-method signature="supervisorsTest()[pri:9, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:08:04 IST" name="supervisorsTest" finished-at="2025-09-17T14:08:15 IST" duration-ms="11211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- supervisorsTest -->
        <test-method signature="qualificationsTest()[pri:10, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:08:15 IST" name="qualificationsTest" finished-at="2025-09-17T14:08:49 IST" duration-ms="34032" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Qualifications test failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Qualifications test failed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.PIMTest.qualificationsTest(PIMTest.java:723)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- qualificationsTest -->
        <test-method signature="membershipsTest()[pri:11, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:08:49 IST" name="membershipsTest" finished-at="2025-09-17T14:08:51 IST" duration-ms="2224" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- membershipsTest -->
        <test-method signature="employeeListSearchTest()[pri:12, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:08:51 IST" name="employeeListSearchTest" finished-at="2025-09-17T14:09:07 IST" duration-ms="15771" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Employee 'Divya Sri Surya' not found in Employee List]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Employee 'Divya Sri Surya' not found in Employee List
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.PIMTest.employeeListSearchTest(PIMTest.java:850)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- employeeListSearchTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.PIMTest@762ef0ea]" started-at="2025-09-17T14:09:07 IST" name="tearDown" finished-at="2025-09-17T14:09:07 IST" duration-ms="382" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.PIMTest -->
    </test> <!-- PIM Tests -->
    <test started-at="2025-09-17T14:09:07 IST" name="Leave Module Tests" finished-at="2025-09-17T14:10:52 IST" duration-ms="104983">
      <class name="com.orangehrm.LeavePage">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:09:07 IST" name="setUp" finished-at="2025-09-17T14:09:15 IST" duration-ms="7636" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="openLeavePage()[pri:1, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:09:15 IST" name="openLeavePage" finished-at="2025-09-17T14:09:20 IST" duration-ms="4542" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openLeavePage -->
        <test-method signature="validateApplyLeavePage()[pri:2, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:09:20 IST" name="validateApplyLeavePage" finished-at="2025-09-17T14:09:22 IST" duration-ms="2680" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateApplyLeavePage -->
        <test-method signature="validateMyLeavePage()[pri:3, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:09:22 IST" name="validateMyLeavePage" finished-at="2025-09-17T14:09:45 IST" duration-ms="22261" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[My Leave Page Test Failed: Expected condition failed: waiting for visibility of element located by By.xpath: //h5[normalize-space()='My Leave'] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: My Leave Page Test Failed: Expected condition failed: waiting for visibility of element located by By.xpath: //h5[normalize-space()='My Leave'] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.LeavePage.validateMyLeavePage(LeavePage.java:108)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMyLeavePage -->
        <test-method signature="addIndividualEntitlement()[pri:4, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:09:45 IST" name="addIndividualEntitlement" finished-at="2025-09-17T14:10:05 IST" duration-ms="20705" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//ul//a[contains(text(),'Add Entitlement')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//ul//a[contains(text(),'Add Entitlement')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.LeavePage.addIndividualEntitlement(LeavePage.java:172)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addIndividualEntitlement -->
        <test-method signature="addMultipleEntitlement()[pri:5, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:10:05 IST" name="addMultipleEntitlement" finished-at="2025-09-17T14:10:27 IST" duration-ms="21792" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Multiple Entitlement Test Failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//ul//a[contains(text(),'Add Entitlement')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Multiple Entitlement Test Failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//ul//a[contains(text(),'Add Entitlement')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.LeavePage.addMultipleEntitlement(LeavePage.java:228)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addMultipleEntitlement -->
        <test-method signature="searchLeaveEntitlements()[pri:6, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:10:27 IST" name="searchLeaveEntitlements" finished-at="2025-09-17T14:10:49 IST" duration-ms="21791" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Leave Entitlement Search Test Failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//a[contains(text(),'Leave Entitlements')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Leave Entitlement Search Test Failed: Expected condition failed: waiting for element to be clickable: By.xpath: //li[.//span[contains(text(),'Entitlements')]]//a[contains(text(),'Leave Entitlements')] (tried for 15 second(s) with 500 milliseconds interval)
Build info: version: '4.21.0', revision: '79ed462ef4'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 140.0.7339.128, chrome: {chromedriverVersion: 140.0.7339.82 (bc93617e21c3..., userDataDir: C:\Users\91967\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:58453}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:58453/devtoo..., se:cdpVersion: 140.0.7339.128, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63ef572e620900f008886074f83c45ed
at org.testng.Assert.fail(Assert.java:111)
at com.orangehrm.LeavePage.searchLeaveEntitlements(LeavePage.java:268)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchLeaveEntitlements -->
        <test-method signature="logoutTest()[pri:7, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:10:49 IST" name="logoutTest" finished-at="2025-09-17T14:10:52 IST" duration-ms="3172" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.LeavePage@623a8092]" started-at="2025-09-17T14:10:52 IST" name="tearDown" finished-at="2025-09-17T14:10:52 IST" duration-ms="392" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.LeavePage -->
      <class name="com.orangehrm.LoginTest">
        <test-method is-config="true" signature="closeSuite()[pri:0, instance:com.orangehrm.LoginTest@23348b5d]" started-at="2025-09-17T14:10:52 IST" name="closeSuite" finished-at="2025-09-17T14:10:53 IST" duration-ms="325" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeSuite -->
      </class> <!-- com.orangehrm.LoginTest -->
    </test> <!-- Leave Module Tests -->
  </suite> <!-- OrangeHRM Automation Suite -->
</testng-results>
